vnext/Microsoft.ReactNative.IntegrationTests/Microsoft.ReactNative.IntegrationTests.vcxproj

```diff
@@ -119,6 +120,7 @@
   <ItemGroup>
     <Manifest Include="Application.manifest" />
     <None Include="packages.config" />
+    <JsBundleEntry Include="AddValues.js" />
```

Comment:
>JsBundleEntry [](start = 5, length = 13)

This is pretty cool! Are we going to have a bundle automatically built for each such entry?
Does VS shows the JS file if it included like this?